antiope-s3/src/Antiope/S3/Messages.hs

```diff
@@ -63,7 +63,7 @@ instance FromJSON S3Message where
 instance ToJSON S3Message where
   toJSON msg = Aeson.object
     [ "eventTime" .= eventTime msg
-    , "eventName" .= T.intercalate ":" ["s3", eventType (eventName msg), action (eventName msg)]
+    , "eventName" .= T.intercalate ":" [eventType (eventName msg), action (eventName msg)]
```

Comment:
Do you mean `eventType` already puts in an `s3:`?

Reply:
Looks like `s3:` shouldn't be there at all.

Reply:
Ah ok.